internal/pkg/manifest/validate.go

```diff
@@ -384,6 +418,24 @@ func (a *AdvancedCount) Validate() error {
 			return fmt.Errorf(`validate "queue_delay": %w`, err)
 		}
 	}
+
+	if err := a.CPU.Validate(); err != nil {
+		return fmt.Errorf(`validate "cpu_percentage": %w`, err)
+	}
+	if err := a.Memory.Validate(); err != nil {
+		return fmt.Errorf(`validate "memory_percentage": %w`, err)
+	}
+	return nil
+}
+
+// Validate returns nil if Percentage is configured correctly.
+func (p *Percentage) Validate() error {
+	if p == nil {
+		return nil
+	}
+	if val := int(*p); val < 0 || val > 100 {
+		return fmt.Errorf("cannot specify %v as Percentage. Must be an integer from 0 to 100", val)
```

Comment:
```suggestion
		return fmt.Errorf("Must be an integer from 0 to 100")
```
I feel like this should suffice to tell the user what's wrong. The error produced would be `validate "cpu_percentage": must be an integer from 0 to 100`.